configuration/configmgr/configmgrlib/mod_template_support/Operation.cpp

```diff
@@ -180,3 +191,35 @@ void Operation::saveAttributeValues(Variables *pInputs, const std::shared_ptr<En
         }
     }
 }
+
+
+void Operation::processNodeValue(Variables *pVariables, const std::shared_ptr<EnvironmentNode> &pEnvNode)
+{
```

Comment:
Should you check for pVariables != nullptr ?

Reply:
Previous comment about a trusted caller applies. However, this method is used during template execution which is wrapped with a try/catch when execution begins.